tests/cypress/support/helpers/wagmi.ts

```diff
@@ -0,0 +1,38 @@
+import { defineChain, http, type Hex } from 'viem'
+import { mainnet } from 'viem/chains'
+import { createWagmiConfig } from '@ui-kit/features/connect-wallet'
+import { createTestConnector } from './test-connector'
+
+/** Configuration options for creating a test Wagmi config */
+type Options = {
+  /** Private key for the test account */
+  privateKey: Hex
+  /** RPC URL for the Ethereum network */
+  rpcUrl: string
+  /** Block explorer URL for the network */
+  explorerUrl?: string
+}
+
+/**
+ * Creates a Wagmi configuration for testing with a private key connector to a custom (testnet) RPC
+ *
+ * @param options - Configuration options
+ * @returns Configured Wagmi config instance for testing
+ */
+export function createTestWagmiConfig({ privateKey, rpcUrl, explorerUrl }: Options) {
+  // Tenderly docs recommends using chain ID 73571 to prevent replay attack, but a lot of our code relies on `if chain === Chain.Ethereum`.
```

Comment:
That's why I always argue we should stop checking specific chains and check for features instead